src/components/header/template.njk

```diff
@@ -91,8 +91,8 @@
               <li class="govuk-header__navigation-item">
                 <a{% if language == 'en' %} aria-current="page"{% else %} class="govuk-header__link"{% endif %}
                   lang="en" hreflang="en" href="{{ params.languageToggle.en.href }}">English</a>
-              </li>
-              <li class="govuk-header__navigation-item">
+		&nbsp;
+              </li><li class="govuk-header__navigation-item">
```

Comment:
the styles for the list items are using display inline-block which means that the list is whitespace sensitive, this fixes an issue where you could see a little white underline like this: `English _  |  Cymraeg` it might be better to adjust the css but that feels like a bigger change. Fixed it with this change because I noticed it was showing up in a lot of screenshots updated by these changes (due to slight text rendering differences with new backstop version)